app_native/app.cpp

```diff
@@ -4,46 +4,46 @@
 #include "apps/i18n.h"
 #include <assert.h>
 
-namespace Sample {
+namespace Ben 10 {
```

Comment:
I think this should be changed back to Sample, haha

Reply:
> I think this should be changed back to Sample, haha

Oh! I don't remember changing that. I must've copied the wrong files. Maybe I'll do a new fork and copy the right files to it. Reminds me of when I was making hot chocolate and put cocoa in the sugar without realising it. My mind slips up sometimes...

Reply:
I'll fix it tomorrow.